ansible/setup-management-network.sh

```diff
@@ -11,7 +11,7 @@ echo
 echo "STEP 1: Checking for j2cli package..."
 if ! command -v j2; then
     echo "j2cli not found, installing j2cli"
-    apt-get install -y j2cli
+    pip install j2cli
```

Comment:
suggest to specify the version here, so that it won't break when j2cli upgrade and introduce some non-backward compatible feature.

Reply:
@stephengzh Can you address the comment from @lguohan?

Reply:
@lguohan Sorry, I missed this comment before.
 "j2cli upgrade": Do you mean python2 upgrades to python3? If so, I think it is much better to specify the version as pip2 here for now.  But if python3 is applied, "pip3 install j2cli" will be applied. Or do you have some better suggestions?

Reply:
I think Guohan's suggestion is to run the pip command with package version specified like below:
```
pip install j2cli==0.3.10
```

Reply:
Oh, I see, so we just specify j2cli version as 0.3.10 to avoid future potential break.

Reply:
Right. Stick with the current working version.